crates/libs/enclaves/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+#![doc = include_str!("../readme.md")]
+#![cfg(windows)]
+#![allow(non_camel_case_types, non_snake_case)]
+
+mod bindings;
+pub use bindings::*;
+
+// The following functions should be generated but are not currently included in the Win32 metadata.
+
+// EnclaveCopyIntoEnclave
+// EnclaveCopyOutOfEnclave
+// EnclaveRestrictContainingProcessAccess
+// InterlockedPushListSList
+// LdrDisableThreadCalloutsForDll
+// LdrResolveDelayLoadedAPI
+// RtlGetLastNtStatus
+// RtlRaiseStatus
+// RtlUnhandledExceptionFilter
```

Comment:
@joshwatson just need to patch these in by hand for now and then we should be ready to take it for a spin. 😊